client/src/pages/Explore.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import moment from 'moment';
+
+import * as ExploreStyles from '../styles/ExploreSyles';
+import ProjectCard from '../components/ProjectCard';
+import Loading from '../components/Loading';
+import { withPageContext } from '../components/pageContext';
+import { OutlinedSelect } from '../components/Inputs';
+import { getUserProjects } from '../api/projects';
+import ProjectGrid from '../components/ProjectGrid';
+
+class Explore extends React.Component {
+  state = {
+    projects: [],
+    industries: [
+      'All Industries',
+      'Technology',
+      'Marketing',
+      'Engineering',
+      'Art',
+      'Film'
+    ],
+    locations: [
+      'Everywhere',
+      'New York',
+      'California',
+      'Georgia',
+      'Florida'
+    ],
+    industrySelect: 'All Industries',
+    locationSelect: 'Everywhere',
+    loading: true
+  };
+
+  async componentDidMount() {
+    const getProjects = await getUserProjects(this.props.userDetails.id);
+    const projects = getProjects.data;
+
```

Comment:
Why not have loading set to false when the data loads instead of using setTimeout? Like: if getProjects.data, set state and set the loading and if not set an error of some sort probably

Reply:
Yes, I did that at first, I want the loading indicator to show at least a little, else it would not show. I feel like having it show for a half a second would hide that pop-in effect a little bit

Reply:
After thinking about it some more I removed the timeout